cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/DocumentRevisionBuilder.java

```diff
@@ -237,7 +237,7 @@ public static DocumentRevision buildRevisionFromMap(URI documentURI, Map<String,
 
         String docId = (String) map.get("_id");
         String revId = (String) map.get("_rev");
-        Boolean deleted = map.get("_deleted") == null ? false : (Boolean) map.get("_deleted");
+        Boolean deleted = map.get("_deleted") == null ? Boolean.FALSE : (Boolean) map.get("_deleted");
```

Comment:
It'd be good to see the failure message the FindBugs is spewing out for each of these fixes.